Microsoft.AzCopy/Microsoft.AzCopy.Linux.Arm64/Microsoft.AzCopy.Linux.Arm64.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Title>Microsoft.AzCopy.Linux.Arm64</Title>
+    <Version>10.21.0</Version>
```

Comment:
is there any way to pull the version here programmatically? or do we need to change this every release?

Reply:
I was looking into this while creating it. I believe you can pull environment variables here, but I didn't wind up experimenting with it enough. We can look into addressing this when we get to automating the creation of this package.